yarn.lock

```diff
@@ -2306,7 +2306,7 @@
   dependencies:
     "@cypress-design/css" "^0.15.1"
 
-"@cypress-design/vue-button@^0.10.2":
+"@cypress-design/vue-button@0.10.2", "@cypress-design/vue-button@^0.10.2":
```

Comment:
Can we use an open range in the package.json so this matches?

Reply:
fixed by 3c7bd2685b8d55dcfc2da9e42c4265e770777e22